### PR TITLE
Fail on asciidoc error and fix one context

### PR DESCRIPTION
**Pull request by lzap**
This patch removes unmatched endif statement which caused asciidoc to print an ERROR.

Because asciidoc still returns zero code, the build continued. The makefile now checks for ERROR strings in asciidoc output and fail if one is found.

This patch also removes one warning - reused id. This is to serve as an example how to fix those. Once we get rid of all warnings we can start failing when warnings are found so everything can be even more strict.

### CODE REVIEW

File: guides/common/modules/proc_installing-the-satellite-server-packages.adoc [hunk 1]
```diff
@@ -43,12 +48,17 @@ endif::[]
...
 
 ifdef::foreman-deb[]
 == [[ubuntu-1804]]Ubuntu 18.04 (Bionic)
 
+:context: ubuntu-bionic
 include::proc_installing-server-packages-deb.adoc[]
 endif::[]
+
+ifdef::parent-context[:context: {parent-context}]
+ifndef::parent-context[:!context
```
Comment (melcorr) [inline guides/common/modules/proc_installing-the-satellite-server-packages.adoc:64]: should this be closed?

Comment (lzap) [reply guides/common/modules/proc_installing-the-satellite-server-packages.adoc]: Yeah weird. Accident.

**Comment by lzap** [comment]
Rebased.

**Comment by lzap** [comment]
Build fails because of two unrecognized UTF-8 codes, I will remove them:

```
asciidoctor: ERROR: failed to parse formatted text: ​<strong>Install</strong> &mdash; to install a new package using the default method. Alternatively, select the drop-down icon to the right of the button and select a method to use. Selecting the <strong>via remote execution - customize first</strong> menu entry will take you to the <strong>Job invocation</strong> page where you can customize the action.
asciidoctor: ERROR: failed to parse formatted text: ​<strong>Update</strong> &mdash; to update an existing package in the host collection using the default method. Alternatively, select the drop-down icon to the right of the button and select a method to use. Selecting the <strong>via remote execution - customize first</strong> menu entry will take you to the <strong>Job invocation</strong> page where you can customize the action.
```

**Comment by lzap** [comment]
For the record, there is a long discussion about implementing `--` and `---` for em/en dashes in asciidoc, steering committee needs to decide because it is a breaking change: https://github.com/asciidoctor/asciidoctor/issues/1174

I am gonna replace these with a minus for now to get builds rolling.

**Review by melcorr — APPROVED** [review]

**Pull request merged**